src/ui/wxWidgets/wxutils.h

```diff
@@ -59,9 +59,25 @@ inline wxString& operator<<(wxString& w, const stringT& s) {
 
```

Comment:
Wouldn't it be more elegant to replace the following 3 with a single
ApplyPrefFont(wxWindow *win, const PWSprefs::StringPrefs fontPref) ?